core/include/cib.h

```diff
@@ -65,6 +65,18 @@ static inline void cib_init(cib_t *__restrict cib, unsigned int size)
     *cib = c;
 }
 
+/**
+ * @brief   Returns the size of a cib_t
```

Comment:
this comments needs rewording IMO, the size of a `cib_t` would be `sizeof(cib_t)`.